site-structure.json

```diff
@@ -34,7 +34,8 @@
                     { "pageName": "Video Content and Learning", "href": "/VideoContentAndLearning.html"},
                     { "pageName": "Audio Content and Learning", "href": "/AudioContentAndLearning.html"},
                     { "pageName": "Authoring of Content", "href": "/AuthoringOfContent.html"},
-                    { "pageName": "Cognitive Considerations", "href": "/CognitiveConsiderations.html"}
+                    { "pageName": "Cognitive Considerations", "href": "/CognitiveConsiderations.html"},
+                    { "pageName": "Web Games and Simulations", "href": "/WebGamesAndSimulations.html"}
```

Comment:
I think this ToC link to the games/sims page should be moved up after audio content and before authoring. It seems geared toward a particular type of content, like video, audio, etc) and so might make more sense near those topics. What do you think, @jhung?